tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -35,3 +35,10 @@ tests:
     extra_args: OVERLAY_CONFIG="overlay-gd32-spi-dma.conf"
     platform_allow: gd32f403z_eval gd32f407v_start gd32f450i_eval gd32f450v_start
       gd32f450z_eval gd32f470i_eval gd32vf103c_starter gd32vf103v_eval longan_nano longan_nano_lite
+  drivers.mcux_flexio_spi.loopback:
+    tags: flexio
+    extra_args: DTC_OVERLAY_FILE="boards/${BOARD}.overlay;overlay-mcux-flexio-spi.overlay"
+    extra_configs:
+      - CONFIG_SPI_MCUX_FLEXIO=y CONFIG_SPI_ASYNC=y
+    filter: CONFIG_MCUX_FLEXIO
+    platform_allow: mimxrt1064_evk
```

Comment:
@MikhailSiomin ping on this- do you see test failures when you remove the `platform_allow`?

Reply:
@danieldegrasse, without `platform_allow`, some other tests break. I removed the commit with flexio driver test because I'm not sure that the complexity of adding this test is justified.